src/settings-compat-v7/_components.scss

```diff
@@ -67,6 +67,12 @@ $bd-alert-basic-success: 0 !default;
 $bdl-alert-basic-success: 4px solid $c-alert-basic-success !default;
 $bgc-alert-basic-success: $c-white !default;
 
+// Badge
+$badge-notification-size: 18px !default;
```

Comment:
to be more consistent with other components, what about name it as `$size-badge-notification` ?

i.e.: https://github.com/SUI-Components/sui-theme/blob/master/src/settings-compat-v7/_components.scss#L122
https://github.com/SUI-Components/sui-theme/blob/master/src/settings-compat-v7/_components.scss#L275